### DISCUSSION TITLE
Ouster-sdk version

### DISCUSSION THREAD

**Discussion by ovaag** [ouster-lidar/ouster-ros#360] [General]
Hi :wave: 

Current master for both ros1 and ros2 is currently using [v0.10.0](https://github.com/ouster-lidar/ouster-sdk/tree/2898060fb47a69b4fbf9807d3c17f6fdc8249379) of ouster-sdk.

I tried building latest master with ouster-sdk `0.12.0` and the build fails. Latest master with ouster-sdk `0.11.1` succeeds, and seem to work on my robot.

Are the newer versions of ouster-sdk adding any value to this ros driver and if so are there plans to bump ouster-sdk in current master?

**Comment by Samahu** [comment]
Hi @ovaag, thanks for your question. For the past two releases we were mostly focused on the python sdk so there is little incentive to update the SDK however I intend to bump the SDK version to 0.11 soon and later 0.12 or 0.13. The reason for not going straight to 0.12 is that it had a fair amount of breaking changes some of which is incompatible with the ouster-ros code, mainly the Point Cloud Customization code. This would require a bit of effort to adjust to the driver to it. But going to 0.11 should be feasible and going to make that upgrade soon.
